apps/nextjs/src/app/(dashboard)/dashboard/_components/dashboard-content-skeleton.tsx

```
@@ -0,0 +1,29 @@
+export default function DashboardContentSkeleton() {
+  return (
+    <div
+      role="status"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<output>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)